sdk/daml-lf/transaction/src/main/protobuf/com/digitalasset/daml/lf/transaction.proto

```diff
@@ -96,15 +96,17 @@ message Versioned {
   bytes payload = 2;
 }
 
-// A self contains representation of a committed contract.
+// A self contained representation of a committed contract.
 message FatContractInstance {
   bytes contract_id = 1;
   string package_name = 2;
-  repeated int32 package_version = 9; // required *since dev(
+  repeated int32 package_version = 9; // required *since dev*
   com.digitalasset.daml.lf.value.Identifier template_id = 3;
   bytes create_arg = 4;
   repeated string non_maintainer_signatories = 5;
   repeated string non_signatory_stakeholders = 6;
+  // Long.MinValue denotes that the creation time is unknown and should be derived from the context
+  // For Create nodes, this must be set to 0.
```

Comment:
Arrghhhh. 
This is all my fault ! :cry:

Reply:
Yeah. I briefly considered to change the value for Create nodes to match "now" until I figured that then I cannot enforce that "now" is only used for relative contract IDs. Moreover, I didn't know what repercussions a change here would have. So I left it as is and added just more tech debt to the pile.